changelog/pending/20230511--sdk-nodejs--replace-empty-interfaces-with-type-aliases.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: chore
+  scope: sdk/nodejs
+  description: Replaces empty interfaces with type aliases. Empty interfaces are equivalent to their supertype; this change expresses these type definitions using type aliases instead of interface extention to provide better clarity. This change will not affect type-checking.
```

Comment:
```suggestion
  description: Replaces empty interfaces with type aliases. Empty interfaces are equivalent to their supertype; this change expresses these type definitions using type aliases instead of interface extension to provide better clarity. This change will not affect type-checking.
```